File: ImagesObjc/ImagesObjc/PhotoCell.m
//
//  PhotoCell.m
//  ImagesObjc
//
//  Created by Bondar Yaroslav on 12/28/18.
//  Copyright © 2018 Bondar Yaroslav. All rights reserved.
//

#import "PhotoCell.h"

@implementation PhotoCell

- (instancetype)initWithFrame:(CGRect)frame {
    self = [super initWithFrame:frame];
    if (self) {
        _imageView = [[UIImageView alloc] init];
        _imageView.contentMode = UIViewContentModeScaleAspectFill;
        _imageView.translatesAutoresizingMaskIntoConstraints = NO;
        _imageView.clipsToBounds = YES;
        _imageView.backgroundColor = [UIColor lightGrayColor];
        _imageView.opaque = YES;
        [self addSubview: _imageView];
    }
    return self;
}

-(void)layoutSubviews {
    [super layoutSubviews];
    _imageView.frame = self.bounds;
}

@end

File: DragDrop/DragDrop/PhotoCell.swift
//
//  PhotoCell.swift
//  DragDrop
//
//  Created by Bondar Yaroslav on 8/6/18.
//  Copyright © 2018 Bondar Yaroslav. All rights reserved.
//

import UIKit

final class PhotoCell: UICollectionViewCell {
    
    @IBOutlet private weak var photoImageView: UIImageView!
    
    func config(with object: UIImage?) {
        photoImageView.image = object
    }
}
